src/params/syntax/async-gen-named-func-expr.template

```diff
@@ -13,7 +13,6 @@ info: |
         7. Let closure be ! AsyncGeneratorFunctionCreate(Normal, FormalParameters,
            AsyncGeneratorBody, funcEnv, strict).
         [...]
-flags: [async]
```

Comment:
all the respective cases using templates from the `syntax` folder have the negative flag, i.e.:

```yaml
negative:
  phase: early
  type: SyntaxError
```